jobserver/forms.py

```diff
@@ -134,6 +134,9 @@ class Meta:
             "repo",
             "branch",
         ]
+        help_texts = {
+            "name": "This will the name of the directory on the backend server(s) in which your research jobs are run.",
```

Comment:
missed a word "be" :) 

Might be a little more user-friendly something like this:
"Enter a descriptive name which makes this workspace easy to identify. It will also be the name of the directory in which you will find results after jobs from this workspace are run."

Are there any requirements like "this must not contain special characters like £$%" or "you can enter spaces but they will be converted to underscores"?

Reply:
Brilliant, thank you!

Yes, there's a validator on the name that ensures only letters, numbers, underscores, or hyphens are allowed.  If you use a character other than those you get an error explaining what is allowed.